### PR TITLE
Go from “Dart webdev” to “Dart for the Web”

### PR DESCRIPTION
**Pull request by filiph**
Update logo and the few places where “webdev” was still used as a ‘brand’.

Fixes part of https://github.com/dart-lang/site-webdev/issues/489.

Staged at https://filiph-webdev-staging.firebaseapp.com/.

cc @lukechurch

### CODE REVIEW

**Comment by filiph** [comment]
Oh, and I optimized the SVG with [SVGOMG](https://jakearchibald.github.io/svgomg/) so that's why there's so many deletions. The current SVGs are woefully bloated.

File: src/_codelabs/angular_components/index.md [hunk 1]
```diff
@@ -24,9 +24,10 @@ To complete this codelab, you need the following:
 *   A web connection and modern browser
 
 This codelab assumes that you are familiar with Dart web app development.
-If you aren’t, instead try one of the [other webdev codelabs](/codelabs).
-Familiarity with AngularDart development isn't required but is helpful,
-since this codelab doesn't explain
+If you aren’t, instead try one of the 
+[other Dart for the Web codelabs](/codelabs).
```
Comment (kwalrath) [inline src/_codelabs/angular_components/index.md]: Dart for the web?

The capitalized Web makes it look all 1993 to me. (Google style is to not capitalize "web" unless you're referring to something specific like "World Wide Web".)

But capitalizing it does help chunk it a bit, making it a little easier to read.)

An alternative might be to just say **other Dart web codelabs** or **web-specific Dart codelabs**.

Or you could justify capitalization by calling "Dart for the Web" a product name. A 1993 product name. :)

I'm conflicted.

File: src/_guides/get-started.md [hunk 2]
```diff
@@ -198,7 +198,7 @@ For a gentler approach, try the codelab
 
 Or check out these resources:
 
-* Dart webdev tutorials and codelabs
+* Dart for the Web tutorials and codelabs
```
Comment (kwalrath) [inline src/_guides/get-started.md]: Same comment as above, except we probably have more flexibility here. Maybe:

**Web-related tutorials and codelabs for Dart:**

**Comment by chalin** [comment]
Looks good.

FYI, there is another occurrence of "webdev" in `site-webdev/src/_codelabs/darrrt/index.md`:
> You can find other codelabs on the ~webdev~ **Dart for the Web** codelabs page.

Maybe you'll also want to change it in `site-webdev/_config.yml`:
> description: "Dart ~webdev~**for the Web** is a platform for building structured web apps. ..."

File: src/_codelabs/angular_components/index.md (hunk 1 above)
Comment (chalin) [inline src/_codelabs/angular_components/index.md]: I like the terseness of "Dart web codelabs".

Comment (filiph) [inline src/_codelabs/angular_components/index.md]: > I like the terseness of "Dart web codelabs".

Me too! I'll use that.

> Dart for the web?

That means I need to update the logo as well, right? It currently uses capitalized "Web"

screen shot 2017-05-05 at 10 04 45

Comment (kwalrath) [inline src/_codelabs/angular_components/index.md]: You have more leeway there because it's a title... but yeah, I'd personally lowercase "web".

Comment (filiph) [inline src/_codelabs/angular_components/index.md]: Ack, done.

File: src/_guides/get-started.md (hunk 2 above)
Comment (filiph) [inline src/_guides/get-started.md]: Ack, done.

**Comment by filiph** [comment]
Done: https://filiph-webdev-staging.firebaseapp.com

I went with "Dart for the web", and put all requested changes in there.

I rewrote the `_config.yml` description slightly because it seemed out of place when using "Dart for the web":

> Dart is a programming language for building high quality apps. This site includes documentation for building such apps for the web.

File: _config.yml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Site settings
 title: webdev.dartlang.org
-description: "Dart webdev is a platform for building structured web apps. It includes a language, a VM, libraries, tools, and compilers to many targets, including JavaScript."
+description: "Dart is a programming language for building high quality apps. This site includes documentation for building such apps for the web."
```
Comment (kwalrath) [inline _config.yml]: I'm not crazy about this description. How about something like this:

Documentation to help you build high-quality web apps, using Dart tools and libraries.

Comment (chalin) [inline _config.yml]: Yes, more concise and a bit clearer.

**Comment by lukechurch** [comment]
LGTM, thanks!

**Comment by kwalrath** [comment]
@filiph, I'll adjust the description in a separate PR (unless you're already working on that).

**Comment by filiph** [comment]
I'm working on that now. Sorry, I missed your note and merged prematurely.

**Pull request merged**